tf2/include/tf2/LinearMath/MinMax.h

```diff
@@ -17,6 +17,8 @@ subject to the following restrictions:
 #ifndef GEN_MINMAX_H
 #define GEN_MINMAX_H
 
+#include <tf2/LinearMath/Scalar.h>
```

Comment:
For better or worse, the style used here (as seen in [QuadWord.h](https://github.com/ros2/geometry2/blob/49d4be4e8dd08282fc3bf5238459ce41d6befe6f/tf2/include/tf2/LinearMath/QuadWord.h#L19) and [Vector3.h](https://github.com/ros2/geometry2/blob/49d4be4e8dd08282fc3bf5238459ce41d6befe6f/tf2/include/tf2/LinearMath/Vector3.h#L21)) seems to be:

```suggestion
#include "Scalar.h"
```

Reply:
quotes in f89398d